doc/source/analyzing/objects.rst

```diff
@@ -591,13 +591,25 @@ This is equivalent to:
 
 .. code-block:: python
 
-   proj = ds.proj(("gas", "density"), "x", data_source=reg, method="mip")
+   proj = ds.proj(("gas", "density"), "x", data_source=reg, method="max")
```

Comment:
should we deprecate `"mip"` ?

Reply:
I saw deprecation messages for "mip" already in certain places, not sure if there isn't one here already or not.

Reply:
Oh that's right, I forgot about it. Nothing to do here !